wvdecrypter/wvdecrypter.cpp

```diff
@@ -33,7 +33,7 @@
 #error  "WIDEVINECDMFILENAME must be set"
 #endif
 
-#define LOCLICENSE 1
+//#define LOCLICENSE
```

Comment:
Leave both uncommented but set to 0

Reply:
sorry, im a c++ noob.
the checks later are
#ifdef LOCLICENSE
To me that just says if it's defined - not check it value?
So, if I set to 0, its still defined isn't it?

Reply:
Yep you're right, sorry!
Just to make it consistent then have them both 1 and commented out. i.e changes are just to add `//`

Reply:
i just changed it to use if's instead, haha.
should i go back?